config/config.go

```diff
@@ -40,9 +42,9 @@ type Crypt struct {
 
 // UMController configuration for update controller.
 type UMController struct {
-	ServerURL string           `json:"serverUrl"`
-	UMClients []UMClientConfig `json:"umClients"`
-	UpdateTTL Duration         `json:"updateTtl"`
+	ServerURL string            `json:"serverUrl"`
+	UMClients []UMClientConfig  `json:"umClients"`
+	UpdateTTL aostypes.Duration `json:"updateTtl"`
```

Comment:
not relevant changes. It already done on another mr

Reply:
Done

Reply:
Done